utils/utils.js

```diff
@@ -15,12 +14,69 @@ const {
   CLIENT_SECRET,
 } = process.env;
 
+/**
+ * @param {NodeResult} release
+ */
+const getVersion = (release) => {
+  /**
+   * Electron releases urls have a well-known format. E.g.:
+   * https://github.com/electron/electron/releases/tag/v14.0.0-beta.2
+   * We only need the latest part (i.e. `14.0.0-beta.2`)
+   */
+  return release.url.split('/v').pop();
+};
+
+/**
+ * Transforms a NodeResult into a Release
+ * @param {NodeResult[]} releases
+ */
+const toReleases = (releases) => {
+  const stables = releases.filter((release) => !release.isPrerelease);
+
+  const versions = stables.map((release) => {
+    return getVersion(release);
+  });
+
+  return versions;
+};
+
+const getReleases = async () => {
+  const query = `
+{
+  repository(owner: "electron", name: "electron") {
+    refs(refPrefix: "refs/tags/", first: 1) {
+      nodes {
+        repository {
+          releases(first: 100, orderBy: {field: CREATED_AT, direction: DESC}) {
```

Comment:
Between 2 major releases there can be up to ~60 nightlies now (one per day), then we have alphas (are they going to dissappear?), betas, minor/patches for all supported majors, etc. Because the list is returned by date and not semver sorted I need to ask for a bunch to make sure I can find the latest one.

Reply:
I think that's fine, although IIRC nightly releases are not published as releases on GitHub, only tagged.

Reply:
You are right, still there are a bunch of alphas and betas.
If we get throthled we might want to revisit this but I'd prefer to keep this number for now.